webapp/components/notifications/Notification/Notification.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <ds-space :class="[{ read: notification.read }, notification]" margin-bottom="x-small">
+    <no-ssr>
+      <ds-space margin-bottom="x-small">
+        <hc-user
+          v-if="resourceType == 'Post'"
+          :user="post.author"
+          :date-time="post.createdAt"
+          :trunc="35"
+        />
+        <hc-user v-else :user="comment.author" :date-time="comment.createdAt" :trunc="35" />
+      </ds-space>
+      <ds-text color="soft">
+        {{ $t('notifications.menu.mentioned', { resource: resourceType }) }}
+      </ds-text>
+    </no-ssr>
+    <ds-space margin-bottom="x-small" />
+    <nuxt-link
+      class="notification-mention-post"
+      :to="{ name: 'post-id-slug', params, ...hashParam }"
+      @click.native="$emit('read')"
+    >
+      <ds-space margin-bottom="x-small">
+        <ds-card
+          :header="post.title || comment.post.title"
+          hover
+          space="x-small"
+          class="notifications-card"
+        >
+          <ds-space margin-bottom="x-small" />
+          <div v-if="resourceType == 'Post'">{{ post.contentExcerpt | removeHtml }}</div>
+          <div v-else>
+            <span class="comment-notification-header">Comment:</span>
+            {{ comment.contentExcerpt | removeHtml }}
+          </div>
+        </ds-card>
+      </ds-space>
+    </nuxt-link>
+  </ds-space>
+</template>
+
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-15T23:09:37Z" title="Friday, August 16th 2019, 1:09:37 am +02:00">Aug 16, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -42,11 +51,26 @@ export default {
   },
   computed: {
     excerpt() {
-      return this.$filters.removeLinks(this.post.contentExcerpt)
+      const excerpt = this.post.id ? this.post.contentExcerpt : this.comment.contentExcerpt
+      return (
+        (!this.post.id ? '<b>Comment: </b>' : '') + excerpt.replace(/<(?:.|\n)*?>/gm, '').trim()
```


Can we implement this behaviour in the template please?
```vue
<div v-if="resourceType == 'Post'">
  {{ contentExcerpt | removeHtml }}
</div>
<b v-else>
  {{ contentExcerpt | removeHtml }}
</b>
```

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-08-16T08:26:35Z" title="Friday, August 16th 2019, 10:26:35 am +02:00">Aug 16, 2019</time>_

---


I don’t really understand this:

- We add `<b>Comment: </b>` there as well.
- Therefore the naming `excerpt` should be changed to `contentInformation`. Which I have already done now!
- In your example a `|` operator is used and I can not find its functionality.
- The if and else case has the same code … ??? But you mean `comment.contentExcerpt` for the second one I suppose …

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-16T08:31:44Z" title="Friday, August 16th 2019, 10:31:44 am +02:00">Aug 16, 2019</time>_

---


Let me explain:
* the `v-else` branch is using `<b>` instead of `<div>`
* the `removeHtml` is a filter that needs to be implemented -it's easy :wink:

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-08-16T10:52:20Z" title="Friday, August 16th 2019, 12:52:20 pm +02:00">Aug 16, 2019</time>_

---


So I found out and succeed. 👍🏼